components/projectsView/ProjectListToggle.js

```diff
@@ -35,4 +35,4 @@ export default function ProjectListToggle(props) {
       </Col>
     </>
   );
-}
+}
```

Comment:
if you add a new line at the end of the file here it will get rid of this red symbol :p https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline